extensions/amp-analytics/0.1/requests.js

```diff
@@ -585,6 +589,9 @@ function reportErrorTemp(url, info) {
     const viewportHeight = Number(viewportSize[1]);
 
     if (viewportWidth == 0 || viewportHeight == 0) {
+      if (Math.random() > REPORTING_THRESHOLD) {
```

Comment:
Suggestion: this could go into the if block above instead

Reply:
😄  I'll leave it as it is since this will go away eventually

Reply:
I figured as much haha